requirements.txt

```diff
@@ -1,7 +1,3 @@
 
-radical.utils>=1.12
```

Comment:
this still would be needed (used in `radical.entk.__init__` module)
p.s. and agree to remove others

Reply:
Well, it is pulled by RP anyway, thus the removal.  But I don't mind keeping it as explicit dependency anyway since it is directly imported...  Will re-add.